core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -305,7 +315,9 @@ public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throw
                 throwable);
           }
         }
-        mMetaStore.putDoraMeta(ufsFullPath, status);
+        if (mMetaStore != null) {
```

Comment:
is it possible line 307 returns a null?

Reply:
In that case, a null is put into the mMetaStore

Reply:
```
307 status = mUfsStatusCache.get(ufsFullPath);
```

The document of this API says:

```
Returns the value associated with key in this cache.
ExecutionError – if an error was thrown while loading the value
```

So `get` will return a valid key if found.
Otherwise it will throw exception.
It will NOT return 'null'.

Thanks to @yyongycy for this detailed inspection of the logic here.